### PR TITLE
fixed issue #22312

### PR DESCRIPTION
**Pull request by Shubham0492**
### Description (*)
When we fetch customer token then it gives an error that this account sign-in was incorrect or your account is disabled temporarily due to there is a check of website id and we were not sending website id in API param. Now I have added website id as param now you have to fetch customer token accordingly website id. 

### Fixed Issues (if relevant)

1. #22312 : magento2 Customer unable to get token in customer/token rest api

### Manual testing scenarios (*)

1. send website id also in the parameter.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Integration/Model/CustomerTokenService.php [hunk 1]
```diff
@@ -84,12 +84,12 @@ public function __construct(
     /**
      * @inheritdoc
      */
-    public function createCustomerAccessToken($username, $password)
+    public function createCustomerAccessToken($username, $password, $websiteId)
```
Comment (dmytro-ch) [inline app/code/Magento/Integration/Model/CustomerTokenService.php]: The `$websiteId` parameter must be optional for backward compatibility.

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hi @Shubham-Webkul,
thank you for your contribution!

There are many unit and integration tests errors. Most of them are caused by backward compatibility issues. Could you please check and resolve them?

File: app/code/Magento/Integration/Model/CustomerTokenService.php (hunk 1 above)
Comment (sidolov) [reply app/code/Magento/Integration/Model/CustomerTokenService.php]: We are not able to change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Comment (Shubham0492) [reply app/code/Magento/Integration/Model/CustomerTokenService.php]: @dmytro-ch thanks for explaination i will update it according to Backward Compatible Development Guide.

Comment (Shubham0492) [reply app/code/Magento/Integration/Model/CustomerTokenService.php]: @dmytro-ch  & @sidolov could you help me to fix Semantic Version Checker build regarding issue. Where will i have to update for fixing this.

Comment (sidolov) [reply app/code/Magento/Integration/Model/CustomerTokenService.php]: Hi @Shubham-Webkul , Semantic Version Checker indicates the problem with the provided solution. Please, check my previous comment

Comment (Shubham0492) [reply app/code/Magento/Integration/Model/CustomerTokenService.php]: @sidolov I have made parameter optional still got an issue. Will i have to create another method. Can be update public method parameters?

**Pull request closed without merging**